src/platform/forms/constants.js

```diff
@@ -18,5 +18,8 @@ export const VA_FORM_IDS = Object.freeze({
   FORM_40_10007: '40-10007',
   FEEDBACK_TOOL: 'FEEDBACK-TOOL',
   FORM_VA_2346A: 'MDOT',
-  FORM_10_10CG: '1010cg',
+  FORM_10_10CG: '10-10CG',
+  FORM_HC_QSTNR: 'HC-QSTNR',
+  FORM_COVID_RESEARCH_VOLUNTEER: 'COVID-RESEARCH-VOLUNTEER',
```

Comment:
`COVID-RESEARCH-VOLUNTEER` should be `COVID-VACCINE-TRIAL` based on schema in `vets-json-schema`

Reply:
changed